x-pack/platform/test/stack_functional_integration/apps/heartbeat/_heartbeat.ts

```diff
@@ -10,7 +10,7 @@ import { FtrProviderContext } from '../../../functional/ftr_provider_context';
 
 export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const retry = getService('retry');
-  const PageObjects = getPageObjects(['common', 'uptime', 'timePicker']);
```

Comment:
removing `uptime` PO to keep the tests under platform (will be moved to QAF) cc @marius-dr